quickwit/quickwit-metastore/src/metastore/grpc_metastore/mod.rs

```diff
@@ -554,7 +554,7 @@ async fn update_channel_endpoints(
     }
 
     for leaving_grpc_address in grpc_addresses_in_use.difference(new_grpc_addresses) {
-        debug!(
+        info!(
```

Comment:
One log line for all leaving servers would be better, as well as using the tracing format: `info!(key=value, "")`.